spark-jobs/src/main/resources/reference.conf

```diff
@@ -80,10 +80,19 @@ spline.mode=BEST_EFFORT
 #
 spline.producer.url="http://localhost:8085/producer"
 
-#Block size in bytes needed for repartition/coalesce
+#possible values: plan, recordCount, schema, dataFrame, schemaSummaries, sample
+
+partition.strategy="plan"
+#Block size in bytes needed for repartition/coalesce, needed for any strategy except for recordCount
 #min.processing.partition.size=31457280
 #max.processing.partition.size=134217728
 
+#if sample is selected
+#partition.sample.size=3
```

Comment:
Considering in statistics the smallest reasonable sample size is 100, it would use at least that, and wouldn't be shy of 1000.